userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -221,7 +209,7 @@ bool cri::resolve(sinsp_container_manager* manager, sinsp_threadinfo* tinfo, boo
 	sinsp_container_info::ptr_t existing_container_info = manager->get_container(container_id);
 
 	if (!existing_container_info ||
-	    existing_container_info->m_metadata_complete == false)
+	    existing_container_info->m_lookup_state == sinsp_container_lookup_state::STARTED)
```

Comment:
this would read better as a guard clause.

Reply:
is it worth logging if this is something other than `sinsp_container_lookup_state::STARTED`